plumgrid_playbooks/roles/plumgrid_gateway/tasks/main.yml

```diff
@@ -26,29 +26,30 @@
 
 - name: Run Gateway Interface Checks
   include: update_gateway.yml
-  when: ifcs_present.stat.exists == True
+  when: ifcs_present.stat.exists
```

Comment:
will give you the same result without `True` no need to check ` True == True => True`